detailed-design/cryptoeconomics.md

```diff
@@ -1,5 +1,5 @@
 ## Cryptoeconomics
-All successful decentralised solutions need a strong incentive mechanism to keep the protocol functioning effectively. On top of that, Obscuro also aims to give its users a good experience by not allowing any form of "MEV". This means it must avoid the gas price bidding war we constantly see in popular blockchains.
+All successful decentralised solutions need a strong incentive mechanism to keep the protocol functioning effectively. 
```

Comment:
let's add the comment about how the requirement is to prevent frontrunning of the actual rollups ( unless it is covered later in the file)